install/.env.dist

```diff
@@ -2,4 +2,3 @@ SERVER_NAME=
 POSTGRES_USER=
 POSTGRES_DB=
 POSTGRES_PASSWORD=
-SERVER_NAME=localhost
```

Comment:
this file should represent all mandatory environment variables, so SERVER_NAME should be present